examples/sample_program.py

```diff
@@ -486,10 +486,13 @@ def parse_images(content):
     return content
 
 
-
+chef = SampleChef()
 if __name__ == '__main__':
     """
     This code will run when the sushi chef is called from the command line.
     """
-    chef = SampleChef()
+
     chef.main()
+
+def construct_channel(*args, **kwargs):
```

Comment:
If you use the "new style" approach to calling the chef script, this shouldn't necessary.